package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ohdsi/atlascharts",
-  "version": "2.0.1",
+  "version": "2.0.2",
```

Comment:
With a new chart implementation, we should increment this by a minor version (2.1.0)

Reply:
Thanks, i fixed it